index.html

```diff
@@ -1795,7 +1795,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-parent-head" scope="row">Superclass Role:</th>
-						<td class="role-parent"><rref>section</rref></td>
+						<td class="role-parent"><rref>phrasingcontent</rref></td>
```

Comment:
This feels strange to me; captions tend to allow flow content, e.g., table or figure captions. (Also, all the examples for the caption role use `div`.)